docs/make.jl

```diff
@@ -0,0 +1,51 @@
+using Documenter
+using Boscia
+
+working_dir = @__DIR__
+
+# Function to copy contents of README.md to index.md
+function copy_readme_to_index()
+    readme_path = joinpath(working_dir, "..", "README.md")  # Adjusted path
+    index_path = joinpath(working_dir, "src", "index.md")
+    readme_content = read(readme_path, String)
+    write(index_path, readme_content)
+end
+
+
+
+# Call the functions to update index.md and 1_algorithms.md
+copy_readme_to_index()
+
+
+# Generate documentation
+makedocs(
+    sitename = "Boscia.jl",
+    modules = [Boscia],
+    format = Documenter.HTML(repolink = "https://github.com/saurabhintoml/Boscia.jl.git"),
```

Comment:
The links have to be changed to the main Boscia repository, i.e. the ZIB-IOL/Boscia one.